### ISSUE TITLE
h5_to_trt.py error

### ISSUE THREAD

**Issue by evil-potato** [preste-ai/camera_ai_whiteboard#1]
Hi!
When I run h5_to_trt.py converted my keras .h5 model to .pb model, the following error occurred.

![1](https://user-images.githubusercontent.com/56706604/116498435-50650000-a8dc-11eb-98b6-810b7a1fc517.png)
![2](https://user-images.githubusercontent.com/56706604/116498440-54911d80-a8dc-11eb-9450-d8b2d63d9c5d.png)
![3](https://user-images.githubusercontent.com/56706604/116498445-58bd3b00-a8dc-11eb-8cdb-8cbe5366e6bf.png)

It seems that there is something wrong with BatchNormalization(name = 'bn_conv1')(x) in my model.
How can I solve this mistake？
I look forward to your reply！
